doc/v2.md

```diff
@@ -0,0 +1,20 @@
+## Guide for Migration from v1 to v2
+
+`v2` contains lots of refactoral changes to improve the performance and configuration by introducing [`@logdna/logger`](https://www.npmjs.com/package/@logdna/logger), [`@logdna/env-config`](https://www.npmjs.com/package/@logdna/env-config), and [`eslint-config-logdna`](https://www.npmjs.com/package/eslint-config-logdna). There are several changes in naming and handling environment variables as well. All environment variables are documented [here](./env.md)
```

Comment:
Suggest that we start the sentence with something like "The v2 release of LogDNA Amazon S3 integration..."or something like that.

Reply:
Done!